djangocms_admin_style/templates/admin/base_site.html

```diff
@@ -7,6 +7,7 @@
 <meta name="viewport" content="width=device-width,initial-scale=1" />
 <link rel="stylesheet" type="text/css" href="{% static 'djangocms_admin_style/css/djangocms-admin.css' %}" />
 <script src="{% static 'djangocms_admin_style/js/datetimefields.js' %}"></script>
+<script src="{% static 'djangocms_admin_style/js/related-widget-wrapper.js' %}"></script>
```

Comment:
yes I agree. Before there was no extra js file included. Do you create a task for this?